src/browser/components/sidebar/browser-sidebar-js.patch

```diff
@@ -1,8 +1,35 @@
 diff --git a/browser/components/sidebar/browser-sidebar.js b/browser/components/sidebar/browser-sidebar.js
-index 4a124003976684e014435854aef69ce29da541d2..61ce44751c36eea3e5ae2ddcc62e42c01459629b 100644
+index e95aa93e912d97aca2f60bf094562ea43a8d179b..166497ed26b075995b2151c5f841dc5dd6d20699 100644
 --- a/browser/components/sidebar/browser-sidebar.js
 +++ b/browser/components/sidebar/browser-sidebar.js
-@@ -578,7 +578,7 @@ var SidebarController = {
+@@ -246,6 +246,8 @@ var SidebarController = {
+   _localesObserverAdded: false,
+   _mainResizeObserverAdded: false,
+   _mainResizeObserver: null,
++  _sidebarResizeObserverAdded: false,
++  _sidebarResizeObserver: null,
+   _ongoingAnimations: [],
+ 
+   /**
+@@ -334,6 +336,17 @@ var SidebarController = {
+       sidebarBox.style.maxWidth = `calc(75vw - ${entry.contentBoxSize[0].inlineSize}px)`;
+     });
+ 
++    if (this._sidebarResizeObserver) {
++      this._sidebarResizeObserver.disconnect();
++    }
++    this._sidebarResizeObserver = new ResizeObserver(async () => {
```

Comment:
I dont think we should add an observer, I believe there are better ways...

Reply:
I found an easier solution by simply moving the splitview-overlay and wrapper into the tabbrowser so the inset % will be correct based on the tabpanels width.

I think it makes more sense for splitview overlay to be nested within the tabpanel so its always the correct size vs. doing extra calculations. I also removed the xhtml file since it seemed to be the occurrence